system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -554,8 +554,7 @@ CLdapSecManager::CLdapSecManager(const char *serviceName, IPropertyTree &config)
 
 CLdapSecManager::~CLdapSecManager()
 {
-    if (!m_cfg->getPropBool("@sharedCache", true))
-        delete m_permissionsCache;
+    m_permissionsCache->Release();
```

Comment:
Might have made more sense to make m_permissionsCached an Owned<CPermissionsCache>